src/routes/sessions/[id]/+page.svelte

```diff
@@ -260,7 +278,13 @@
 						{/key}
 
 						<div class="flex w-full">
-							<ButtonSubmit {handleSubmit} hasMetaKey={isPromptFullscreen} disabled={!prompt}>
+							<ButtonSubmit
+								{handleSubmit}
+								hasMetaKey={isPromptFullscreen}
+								disabled={!prompt ||
+									!$settingsStore?.ollamaModels.length ||
+									!$settingsStore?.ollamaModel}
```

Comment:
Do you know if there could there be a situation where `$settingsStore.ollamaModel` is defined but `$settingsStore?.ollamaModels.length == []` isn't?

Reply:
Sorry for the delay.

Currently, if you are connected to the server, then choose a model, and then disconnect, `ollamaModels` will be reseted to `[]` when you change the session, but `ollamaModel` will still be defined as your previous chosen model.

I think this behavior is correct because if I have my model chosen and I get disconnected from the server, I'd like to have the same model chosen when I get connected again. What do you think?

Reply:
Oh, right. Good point!